Gemfile

```diff
@@ -4,6 +4,7 @@ gem 'sinatra'
 gem 'activerecord', :require => 'active_record'
 gem 'sinatra-activerecord', :require => 'sinatra/activerecord'
 gem 'rake'
+gem 'nokogiri'
```

Comment:
There shouldn't be any scraping going on in this app, so you won't need nokogiri.